README.md

```diff
@@ -1,49 +1,30 @@
-<p align="center">
-  <a href="https://wasmer.io" target="_blank" rel="noopener">
-    <img width="300" src="https://raw.githubusercontent.com/wasmerio/wasmer/master/assets/logo.png" alt="Wasmer logo">
-  </a>
-</p>
-
-<p align="center">
-  <a href="https://spectrum.chat/wasmer">
-    <img src="https://withspectrum.github.io/badge/badge.svg" alt="Join the Wasmer Community"></a>
-  <a href="https://rubygems.org/gems/wasmer">
-    <img src="https://img.shields.io/gem/v/wasmer.svg" alt="Gem"></a>
-  <a href="https://rubygems.org/gems/wasmer">
-    <img src="https://img.shields.io/gem/dt/wasmer.svg" alt="Number of gem downloads"></a>
-  <a href="https://github.com/wasmerio/wasmer/blob/master/LICENSE">
-    <img src="https://img.shields.io/github/license/wasmerio/wasmer.svg" alt="License"></a>
-</p>
-
-Wasmer is a Ruby library for executing WebAssembly binaries:
-
- * **Easy to use**: The `wasmer` API mimics the standard WebAssembly API,
- * **Fast**: `wasmer` executes the WebAssembly modules as fast as possible,
- * **Safe**: All calls to WebAssembly will be fast, but more
-   importantly, completely safe and sandboxed.
-
-# Install
-
-To install the `wasmer` Ruby gem, just run this command in your shell:
+# <img height="48" src="https://wasmer.io/static/icons/favicon-96x96.png" alt="Wasmer logo" valign="middle"> Wasmer Ruby [![Wasmer Ruby Gem version](https://img.shields.io/gem/v/wasmer.svg)](https://rubygems.org/gems/wasmer) [![Wasmer Ruby Gem Documentation](https://img.shields.io/badge/docs-read-green)](https://www.rubydoc.info/gems/wasmer/) [![Wasmer Ruby Gem downloads](https://img.shields.io/gem/dt/wasmer.svg)](https://rubygems.org/gems/wasmer) [![Wasmer Slack Channel](https://img.shields.io/static/v1?label=chat&message=on%20Slack&color=green)](https://slack.wasmer.io)
 
-```sh
-$ gem install wasmer
```

Comment:
Why removing this?

Reply:
Mistake! I will add it back 👍 
Good catch!

Reply:
Fixed!